content/projects/teachopencadd.md

```diff
@@ -0,0 +1,95 @@
+---
+title: TeachOpenCADD
+date: 2020-03-03
+publishdate: 2020-03-03
+nav: false
+weight: 1
+menu:
+    main:
+        parent: Education
+external_resources:
+- name: TeachOpenCADD for Jupyter
+  link: https://github.com/volkamerlab/TeachOpenCADD
+  icon: github
+  more: Jupyter notebooks on computer-aided drug design tasks using open resources
+- name: TeachOpenCADD for KNIME
+  link: https://hub.knime.com/volkamerlab/spaces/Public/latest/TeachOpenCADD/TeachOpenCADD
+  more: KNIME workflows on computer-aided drug design tasks using open resources
+people:
+- key: dominique.sydow
+- key: jaime.rodriguez
+- key: michele.wichmann
+- key: andrea-lilian.morger
+- key: maximilian.driller
+- key: andrea.volkamer
+collaborators:
+- name: Gregory Landrum
+  more: KNIME AG
+- name: Daria Goldmann
+  more: KNIME GmbH
+funding:
+- name: Bundesministerium für Bildung und Forschung, grant ID 031A262C
+- name: Deutsche Forschungsgemeinschaft (DFG), grant ID VO 2353 / 1-1
+- name: HaVo-Stiftung, Ludwig-shafen, Germany
+- name: Open Access Publication Fund of Charité – Universitätsmedizin Berlin
+- name: Stiftung Charité (Einstein BIH Visiting Fellow Project)
+- name: “SUPPORT für die Lehre” program (Förderung innovativer Lehrvorhaben) of Freie Universität Berlin.
+publications:
+- teachopencaddknime
+- teachopencadd
+# You can use this keyword for the brief introductions to the article in category listings
+# This will replace the first 70 words found in the main article. use | <newline> to use multiline strings!
+summary: |
+  TeachOpenCADD is a teaching platform offering tutorials on central topics in cheminformatics and structural bioinformatics.
+  The tutorials contain theoretical background and practical implementations using open source data and software.
+  Implementations are available in two formats.
```

Comment:
Just checking in: In the other projects that I am reviewing (e.g. ratar and kissim), there is not such a `summary` tag. Is that on purpose?

Reply:
@dominiquesydow the summary tag was only needed if the summary that we want to show is not the first text that appears in the project - per default the first few lines until a specific count are used - note sometimes you will see a <!--more--> comment then everything up to there will be considered as summary.